src/edu/gatech/cs2340/trydent/ContinuousEvent.java

```diff
@@ -0,0 +1,59 @@
+package edu.gatech.cs2340.trydent;
+
+/**
+ * Represents an event that is called by the engine every frame.
+ * @author Garrett Malmquist
+ *
+ */
+public abstract class ContinuousEvent {
+	
+	private boolean started = false;
+	
+	public ContinuousEvent() {
+		TrydentEngine.addContinuousEvent(this);
+	}
+	
+	/**
+	 * Called by the engine on update.
+	 */
+	protected final void doUpdate() {
```

Comment:
Do we want protected or package access for this one? I think package access would make more sense, to rid any confusion between this method and onUpdate(). Unless you expect that we (or external users) would want to subclass ContinuousEvent and also override this method.